jekyll/_cci2/getting-started.md

```diff
@@ -54,9 +54,13 @@ The `- image: circleci/ruby:2.4.1` text tells CircleCI what Docker image to use
 
 2. Next, you will be given the option of *following* any projects you have access to that are already building on CircleCI (this would typically apply to developers connected to a company or organization's GitHub account). On the next screen, you'll be able to add the repo you just created as a new project on CircleCI.
 
-3. To add your new repo, ensure that your GitHub account is selected in the dropdown in the upper-left, Select the Add Projects page, and find the repository you just created in the list, then click the **Set Up project** button next to it. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list.png)
+3. To add your new repo, ensure that your GitHub account is selected in the dropdown in the upper-left, Select the Add Projects page, and find the repository you just created in the list (you can also search for it), then click the **Set Up Project** button next to it. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list.png)
 
-4. On the next screen, you're given some options for configuring your project on CircleCI. Leave everything as-is for now and just click the **Start building** button a bit down the page on the right. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-2.0-setup-project-circle101.png) ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-2.0-start-building.png)
+4. On the following screen, choose a language from the dropdown to get a pre-populated config.yml file with suggested best-practices for your project. If the language you are using is not in the list, click the "Use Hello World" button at the bottom of the list. Copy the YAML configuration that appears and save it as a file called "config.yml" in a folder named `.circleci` in the root of your repository's directory.
+
+![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list-2.png)
```

Comment:
If you put the image ref at the end of line 59 after the text it will appear in line with step 4, and then step 5 will be labeled correctly. Currently step 5 is showing as `1.` for me.

Reply:
Also just checking you wanted the images in this section to only reflect the new UI? I think that's pretty much OK as the actual steps are largely the same - apart from where you say select the language from the dropdown - in the old UI they are buttons.

Reply:
Also comment for lines 135 and 137 - the first image in these steps is new UI and the second is old - maybe change to both be new UI to show the workflows map? (sorry I can't add comments to places in the doc where you haven't already made changes for some reason)